lavalink/README.md

```diff
@@ -97,20 +97,19 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
 }
 ```
 
-[`Lavalink::process`]: client/struct.Lavalink.html#method.process
 [Lavalink]: https://github.com/Frederikam/Lavalink
 [`async-tungstenite`]: https://crates.io/crates/async-tungstenite
 [`http`]: https://crates.io/crates/http
 [`rustls`]: https://crates.io/crates/rustls
-[client]: client/struct.Lavalink.html
+[client]: Lavalink
 [discord badge]: https://img.shields.io/discord/745809834183753828?color=%237289DA&label=discord%20server&logo=discord&style=for-the-badge
 [discord link]: https://discord.gg/7jj8n7D
 [github badge]: https://img.shields.io/badge/github-twilight-6f42c1.svg?style=for-the-badge&logo=github
 [github link]: https://github.com/twilight-rs/twilight
 [license badge]: https://img.shields.io/badge/license-ISC-blue.svg?style=for-the-badge&logo=pastebin
 [license link]: https://github.com/twilight-rs/twilight/blob/trunk/LICENSE.md
-[node]: node/struct.Node.html
-[process]: client/struct.Lavalink.html#method.process
+[node]: Node
+[process]: Lavalink::process
```

Comment:
These won't render in the Github Readme, only rustdoc. Need to figure out something here. 

![https://kylie.is-pretty.cool/8AgvpLv.png](https://kylie.is-pretty.cool/8AgvpLv.png)

Reply:
They do not work currently either, which was my reason for passing up on them.